File: displays/blue/blinkOneLED.py
#!/usr/bin/env python3
import Adafruit_BBIO.GPIO as GPIO
import time

LED = "USR0"
delay = 0.25

GPIO.setup(LED, GPIO.OUT)

while True:
    GPIO.output(LED, 1)
    time.sleep(delay)
    GPIO.output(LED, 0)
    time.sleep(delay)

File: displays/blue/blinkOneLED.js
#!/usr/bin/env node
var b = require('bonescript');

var led = "USR0"; 

// Set direction to output
b.pinMode(led, b.OUTPUT);

// Turn LED off
var state = 0;
b.digitalWrite(led, state);

// Every 500ma call toggle()
setInterval(toggle, 500);

function toggle() {
    if(state == 0) {
        state = 1;
    } else {
        state = 0;
    }
    b.digitalWrite(led, state);
}
